java/ql/lib/ext/org.hibernate.internal.util.collections.model.yml

```
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: summaryModel
+    data:
+      - ["org.hibernate.internal.util.collections", "Stack", True, "push", "(Statement)", "", "Argument[0]", "Argument[this]", "taint", "ai-manual"]
```

This model doesn't seem correct, does it? [`Stack`](https://docs.jboss.org/hibernate/orm/6.2/javadocs/org/hibernate/internal/util/collections/Stack.html) is a parameterized type, and the type of the parameter of [`push`](https://docs.jboss.org/hibernate/orm/6.2/javadocs/org/hibernate/internal/util/collections/Stack.html#push(T)) is `T`, which we should see as `Object`, not `Statement`.

Since this is a stack, I also think we could model this as a container, so something like:

```suggestion
      - ["org.hibernate.internal.util.collections", "Stack", True, "push", "(Object)", "", "Argument[0]", "Argument[this].Element", "value", "ai-manual"]
```

In any case, the package name (`internal`) makes me think this is not user-oriented, and we could probably ignore it.